docs/man/rpm.8.scd

```diff
@@ -43,106 +42,70 @@ scripts, file attributes, and descriptive information about the package.
 software to be installed, and source packages, containing the source
 code and recipe necessary to produce binary packages.
 
-One of the following basic modes must be selected: *Query*,
-*Verify*, *Install/Upgrade/Freshen/Reinstall*, *Uninstall*, *Set
-Owners/Groups*, *Show Querytags*, and *Show Configuration*.
-
-# GENERAL OPTIONS
-
-These options can be used in all the different modes.
-
-*-?*, *--help*
-	Print a longer usage message than normal.
-
-*--version*
-	Print a single line containing the version number of *rpm* being
-	used.
-
-*--quiet*
-	Print as little as possible - normally only error messages will be
-	displayed.
-
-*-v*, *--verbose*
-	Print verbose information - normally routine progress messages will
-	be displayed.
-
-*-vv*
-	Print lots of ugly debugging information.
-
-*--rcfile* _FILELIST_
-	Replace the default list of configuration files to be read with
-	_FILELIST_.  See *rpmrc Configuration* for details.
-
-*--load* _FILE_
-	Load an individual macro file.
+# OPERATIONS
+*-e*,
```

Comment:
At the risk of sounding nitpicky, how about listing these operations in the order that they're listed in SYNOPSIS? E.g. erasing surely isn't the most interesting operation :laughing:

Reply:
Right - the idea was to put them in alphabetical order, as per the template. I even started shuffling them that way in SYNOPSIS but at least for the rpm main modes, that doesn't work very well. Which brings us back here. I don't know really.

Reply:
I changed the order to match SYNOPSIS now, including the subsections. And while at it, realize the subsections were all uppercase, so fixed to use mixed.

The subsections seem kinda repetitive when they're now in both synopsis and operations, but otoh there are so many things there it'd probably look chaotic without them :thinking:

Reply:
Yeah you'd want to say "transaction" or such there, the title is pretty terrible. But yep, a "transaction" wont mean much to a newcomer.

Then there's the --restore operation which is this weird cross between a transaction and a query... and should actually be listed under transactions and the synopsis is wrong too because it does accept transaction options, even if not all of them are relevant. Another fixup coming up...

Reply:
I actually added "reinstalling" to that title in this PR but that makes the title much worse, and still doesn't cover everything it does. So think I'll just revert that back: "Installing, updating and erasing packages" gets you enough of the idea I think.

Reply:
I reverted it to the original version of "installing, upgrading and removing" - it's indeed far more reasonable looking, even if not 100% accurate. Lets just go with that for now.

Reply:
Ack, that's already better indeed.